ethereum/smart-contract/test/gatewayToken.test.ts

```diff
@@ -153,33 +176,6 @@ describe('GatewayToken', async () => {
     });
   });
 
-  describe('Gatekeeper Networks', async () => {
-    it('Get gatekeeper network by id', async () => {
-      let network = await gatewayToken.getNetwork(gkn1);
-      expect(network).to.equal('Test GKN 1');
-    });
-
-    it('rename gatekeeper network - reverts if unauthorized', async () => {
-      await expect(gatewayToken.connect(alice).renameNetwork(gkn1, 'Test GKN 1 Renamed')).to.be.revertedWithCustomError(
-        gatewayToken,
-        'Common__Unauthorized',
-      );
-    });
-
-    it('rename gatekeeper network - reverts if network does not exist', async () => {
-      await expect(
-        gatewayToken.connect(alice).renameNetwork(11111, 'Test GKN 1 Renamed'),
-      ).to.be.revertedWithCustomError(gatewayToken, 'GatewayToken__NetworkDoesNotExist');
-    });
-
-    it('rename gatekeeper network', async () => {
-      const newName = 'Test GKN 1 Renamed';
-      await gatewayToken.connect(identityCom).renameNetwork(gkn1, newName);
-      let network = await gatewayToken.getNetwork(gkn1);
-      expect(network).to.equal(newName);
-    });
-  });
```

Comment:
Test moved into network contract test, except for renaming networks. Currently there is no way to update the name of a network aside from deleting the network and creating a new one.